components/scream/scripts/cf-xml-to-yaml

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env python
+
+"""
+Given an XML file containing the CF conventions for standardized field names
+(https://cfconventions.org/standard-names.html), this tool generates a YAML
+file with the same information.
+
+Usage: cf-xml-toyaml <cf-xml-file>
```

Comment:
nitpicky, but should the text read `cf-xml-to-yaml`?  Notice the missing hyphen.

Reply:
Whoops! Nice catch.